lib/Message.php

```diff
@@ -75,12 +75,11 @@ function getBodyAsString() {
             return '';
         }
         $contentLength = $this->getHeader('Content-Length');
-        if (null === $contentLength) {
-            return stream_get_contents($body);
-        } else {
+        if (is_int($contentLength) || ctype_digit($contentLength)) {
```

Comment:
The cast to int would turn an empty string into a 0, and instead I want an invalid value to be ignored altogether.